pages/collections/[slug]/manage/contributions.tsx

```diff
@@ -21,7 +21,6 @@ function CollectionContributions() {
       {({ data }) => (
         <CollectionLayout showSidebar data={data?.collection}>
           <CollectionContributionList<Query>
-            hideHeader={true}
```

Comment:
## Child page headers
Now that we can choose what style of header to use in a child page, the header does not need to be hidden inside model lists